ui/app/routes/vault/cluster/access/method/item/list.js

```diff
@@ -4,7 +4,7 @@ import { singularize } from 'ember-inflector';
 import ListRoute from 'vault/mixins/list-route';
 
 export default Route.extend(ListRoute, {
-  wizard: service(),
+  store: service(),
```

Comment:
The store injection was already present on 1.13+ so perhaps this was not required to inject on routes previous to Ember4?

Reply:
Yes that's correct. Ember Data was automatically injecting it behind the scenes which has since been removed.